package.json

```diff
@@ -33,13 +33,14 @@
     "find-process": "^1.1.0",
     "forever-monitor": "^1.7.1",
     "globby": "^8.0.0",
-    "node-cmd": "^3.0.0",
     "jasmine-reporters": "^2.2.0",
     "platform": "^1.3.5",
     "protractor": "^5.1.0",
     "wait-on": "^2.0.2"
   },
   "bin": {},
-  "dependencies": {},
+  "dependencies": {
```

Comment:
`eslint` complained that this needs to be in `dependencies` section as it is being used in `set-env.js`